### PR TITLE
Clarified version information for IAM roles.

### PR DESCRIPTION
**Pull request by kwcanuck**
Added edits to satisfy the requirements of issue #578.

### CODE REVIEW

File: software_versioned_docs/version-0.27/integrate-iam.md [hunk 1]
```diff
@@ -23,8 +23,9 @@ A few clarifying notes:
 
 * [The Astronomer CLI](cli-quickstart.md)
 * Admin access on an Astronomer Workspace
-* Direct access to your Kubernetes Cluster (e.g. permission to run `$ kubectl describe po`)
-* A compatible version of Kubernetes as described in Astronomer's [Version Compatibility Reference](version-compatibility-reference.md)
+* Direct access to your Kubernetes Cluster with permission to run `$ kubectl describe po`
+* A compatible version of Kubernetes. See [Version Compatibility Reference](version-compatibility-reference.md)
+* Astronomer Software version 0.27.1 or later
```
Comment (jwitz) [inline software_versioned_docs/version-0.27/integrate-iam.md:28]:
```suggestion
* Astronomer Software `v0.27.1` or later
```

File: software_versioned_docs/version-0.26/integrate-iam.md [hunk 2]
```diff
@@ -23,8 +23,9 @@ A few clarifying notes:
 
 * [The Astronomer CLI](cli-quickstart.md)
 * Admin access on an Astronomer Workspace
-* Direct access to your Kubernetes Cluster (e.g. permission to run `$ kubectl describe po`)
-* A compatible version of Kubernetes as described in Astronomer's [Version Compatibility Reference](version-compatibility-reference.md)
+* Direct access to your Kubernetes Cluster with permission to run `$ kubectl describe po`
+* A compatible version of Kubernetes. See [Version Compatibility Reference](version-compatibility-reference.md)
+* Astronomer Software version 0.26.6 or later
```
Comment (jwitz) [inline software_versioned_docs/version-0.26/integrate-iam.md:28]:
```suggestion
* Astronomer Software v0.26.6 or later 
```

**Review by jwitz — APPROVED** [review]
LGTM! Thanks for the quick fix @kwcanuck

**Comment by kwcanuck** [comment]
@jwitz can I ask why you have recommended to truncate _version_ to _v_? This could be a potential accessibility and localization issue.  Just curious.

**Comment by jwitz** [comment]
@kwcanuck y'know, it's just something we've done for a while for no reason other than that's how we've done it. If you think this way is better for accessibility, then I'm all for changing it and adding that detail to our style guide!

File: software_versioned_docs/version-0.27/integrate-iam.md (hunk 1 above)
Comment (kwcanuck) [reply software_versioned_docs/version-0.27/integrate-iam.md:28]: Will move forward with _version_.

**Pull request merged**